### PR TITLE
RDW6, req1: Updates for subject in embargo audit log.

### PR DESCRIPTION
**Pull request by gregcharles**

### CODE REVIEW

File: migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
+-- v2.4.0_4 flyway script
```
Comment (malaffoon) [inline migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:1]: V2.4.0_5?

File: migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    ADD CONSTRAINT fk__audit_district_embargo__subject FOREIGN KEY (subject_id) REFERENCES subject(id),
+    ADD INDEX idx__audit_district_embargo__subject (subject_id),
+    ADD CONSTRAINT fk__audit_district_embargo__district FOREIGN KEY (district_id) REFERENCES district(id),
+    ADD INDEX idx__audit_district_embargo__district (district_id);
+
+DROP TRIGGER trg__district_embargo__insert;
+DROP TRIGGER trg__district_embargo__update;
+DROP TRIGGER trg__district_embargo__delete;
+
+CREATE TRIGGER trg__district_embargo__insert
```
Comment (malaffoon) [inline migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:21]: I don't see this joining in the audit setting. Did we discuss whether this should honor that setting like the other audit tables?

File: migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql (hunk 1 above)
Comment (gregcharles) [reply migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:1]: 👍

File: migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql (hunk 2 above)
Comment (gregcharles) [reply migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:21]: Do you mean the addition of the foreign keys? I need the subject and district names in the audit log, but should I do it a different way?

Comment (malaffoon) [reply migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:21]: If you look in the other triggers, you'll see they join the settings table and only insert the audit records if the audit flag is enabled.

Comment (gregcharles) [reply migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:21]: Ah, got it. I didn't know that. The embargo audit triggers weren't like that before, but I can make them like that now. Should I? I don't know why they would be different, but maybe they just predate the settings thing and weren't retrofitted.

Comment (malaffoon) [reply migrations/warehouse/sql/V2_4_0_5__embargo_audit.sql:21]: I could see an argument either way. But, if they weren't like that before, let's not complicate it.

**Pull request merged**